fairlearn/metrics/_group_metric_set.py

```diff
@@ -12,12 +12,7 @@
 from . import group_miss_rate, group_precision_score, group_r2_score
 from . import group_recall_score, group_roc_auc_score, group_root_mean_squared_error
 from . import group_selection_rate, group_specificity_score, group_zero_one_loss
-
-_GROUP_NAMES_MSG = "The sensitive_feature_names property must be a list of strings"
-_METRICS_KEYS_MSG = "Keys for metrics dictionary must be strings"
-_METRICS_VALUES_MSG = "Values for metrics dictionary must be of type GroupMetricResult"
-
-_ARRAYS_NOT_SAME_LENGTH = "Lengths of y_true, y_pred and sensitive_features must match"
+from ._input_manipulations import _convert_to_ndarray_and_squeeze
```

Comment:
aren't the other methods except for the last one also input manipulations? Should they live there?

Reply:
You mean `_process_predictions()` etc.? I think they're all more specialised.